diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -558,6 +558,7 @@ controller_interface::CallbackReturn DiffDriveController::configure_side(
   }
 
   // register handles
+  registered_handles.clear();
```

Comment:
should this happen on deactivate?

Reply:
That is being executed on activate, before creating the handles.
I can change it to use the other solution if you think it's better.